src/pages/iou/steps/IOUAmountPage.js

```diff
@@ -236,6 +241,7 @@ class IOUAmountPage extends React.Component {
                         value={formattedAmount}
                         placeholder={this.props.numberFormat(0)}
                         keyboardType={CONST.KEYBOARD_TYPE.NUMBER_PAD}
+                        blurOnSubmit={this.state.amount !== ''}
```

Comment:
@sobitneupane can we set `blurOnSubmit` to be `true` always?

Reply:
We are trying to make blurOnSubmit `false` when there is no amount to proceed to next step.
If You mean to keep it always `false`, I am not sure. I avoid it thinking it might cause regression.

Reply:
Let's not add extra logic if it isn't needed. Please see if we can keep it always false without a regression

Reply:
Tested and pushed the change. Working fine in my side. Please test it in your side as well.

Reply:
Thanks for testing it